ScoutSuite/utils.py

```diff
@@ -90,4 +87,26 @@ def is_throttled(e):
     :param e:                           Exception raised
     :return:                            True if it's a throttling exception else False
     """
-    return True if  (hasattr(e, 'response') and 'Error' in e.response and e.response['Error']['Code'] in [ 'Throttling', 'RequestLimitExceeded', 'ThrottlingException' ]) else False
+    return True if (hasattr(e, 'response')
```

Comment:
```suggestion
    return (hasattr(e, 'response') and 'Error' in e.response
                and e.response['Error']['Code'] in ['Throttling', 'RequestLimitExceeded', 'ThrottlingException'])
```
If  you apply the suggestion, make sure that not just the first line is replaced.